test/ecto/schema_test.exs

```diff
@@ -21,13 +21,15 @@ defmodule Ecto.SchemaTest do
   end
 
   test "schema metadata" do
-    assert Schema.__schema__(:source)             == "my schema"
-    assert Schema.__schema__(:prefix)             == nil
-    assert Schema.__schema__(:fields)             == [:id, :name, :email, :password, :count, :array, :uuid, :query_excluded_field, :comment_id]
-    assert Schema.__schema__(:query_fields)       == [:id, :name, :email, :password, :count, :array, :uuid, :comment_id]
-    assert Schema.__schema__(:read_after_writes)  == [:email, :count]
-    assert Schema.__schema__(:primary_key)        == [:id]
-    assert Schema.__schema__(:autogenerate_id)    == {:id, :id, :id}
+    assert Schema.__schema__(:source)              == "my schema"
+    assert Schema.__schema__(:prefix)              == nil
+    assert Schema.__schema__(:fields)              == [:id, :name, :email, :password, :count, :array, :uuid, :query_excluded_field, :comment_id]
+    assert Schema.__schema__(:virtual_fields)      == [:temp]
+    assert Schema.__schema__(:query_fields)        == [:id, :name, :email, :password, :count, :array, :uuid, :comment_id]
+    assert Schema.__schema__(:read_after_writes)   == [:email, :count]
+    assert Schema.__schema__(:primary_key)         == [:id]
+    assert Schema.__schema__(:autogenerate_id)     == {:id, :id, :id}
+    assert Schema.__schema__(:autogenerate_fields) == [:name, :uuid]
```

Comment:
Since we are rewriting it all, can we stop aligning the ==, like the formatted would? Thanks! :)

Reply:
Sounds good to me! Would you prefer me to format the rest of the assertions in that test like the formatter would, or revert them to how they were aligned previously to minimize the diff?

Reply:
Went with the first for now, but happy to rebase if you meant the second.